test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/smoke/ClusteredWorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioIntegrationTest.java

```diff
@@ -32,19 +34,54 @@
 import org.kie.cloud.maven.constants.MavenConstants;
 import org.kie.cloud.tests.common.AbstractCloudIntegrationTest;
 import org.kie.cloud.tests.common.ScenarioDeployer;
+import org.kie.cloud.tests.common.client.util.Kjar;
+import org.kie.cloud.tests.common.client.util.WorkbenchUtils;
+import org.kie.server.api.model.KieContainerStatus;
+import org.kie.server.api.model.KieServerInfo;
+import org.kie.server.client.KieServicesClient;
+import org.kie.server.controller.client.KieServerControllerClient;
 
 @Category({Smoke.class, JBPMOnly.class})
 public class ClusteredWorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioIntegrationTest extends AbstractCloudIntegrationTest {
 
     private static ClusteredWorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenario deploymentScenario;
 
+    private static final String HELLO_RULES_CONTAINER_ID = "helloRules";
+    private static final String DEFINITION_PROJECT_CONTAINER_ID = "definition-project";
+    private static final String CLOUDBALANCE_CONTAINER_ID = "cloudbalance";
+
     @BeforeClass
     public static void initializeDeployment() {
         deploymentScenario = deploymentScenarioFactory.getClusteredWorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder()
                                                       .withExternalMavenRepo(MavenConstants.getMavenRepoUrl(), MavenConstants.getMavenRepoUser(), MavenConstants.getMavenRepoPassword())
                                                       .build();
         deploymentScenario.setLogFolderName(ClusteredWorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioIntegrationTest.class.getSimpleName());
         ScenarioDeployer.deployScenario(deploymentScenario);
+
+        // Workaround to speed test execution.
+        // Create all containers while Kie servers are turned off to avoid expensive respins.
+        KieServerControllerClient kieControllerClient = KieServerControllerClientProvider.getKieServerControllerClient(deploymentScenario.getWorkbenchRuntimeDeployment());
+        KieServicesClient kieServerOneClient = KieServerClientProvider.getKieServerClient(deploymentScenario.getKieServerOneDeployment());
+        KieServicesClient kieServerTwoClient = KieServerClientProvider.getKieServerClient(deploymentScenario.getKieServerTwoDeployment());
+        KieServerInfo serverOneInfo = kieServerOneClient.getServerInfo().getResult();
+        KieServerInfo serverTwoInfo = kieServerTwoClient.getServerInfo().getResult();
+
+        deploymentScenario.getKieServerOneDeployment().scale(0);
+        deploymentScenario.getKieServerTwoDeployment().scale(0);
+        deploymentScenario.getKieServerOneDeployment().waitForScale();
+        deploymentScenario.getKieServerTwoDeployment().waitForScale();
+
+        WorkbenchUtils.saveContainerSpec(kieControllerClient, serverOneInfo.getServerId(), serverOneInfo.getName(), HELLO_RULES_CONTAINER_ID, "hello-rules-alias", Kjar.HELLO_RULES_SNAPSHOT, KieContainerStatus.STARTED);
```

Comment:
As it seems the same thing is done on both servers, maybe factorize in a stream forEach ?

Reply:
PR adjusted, please take another look.

Reply:
ok for me